xl_auth/collection/models.py

```diff
@@ -22,18 +22,20 @@ class Collection(SurrogatePK, Model):
     replaces = Column(db.String(255))
     replaced_by = Column(db.String(255))
 
-    modified_at = Column(db.DateTime, default=dt.datetime.utcnow, onupdate=dt.datetime.utcnow)
-    created_at = Column(db.DateTime, nullable=False, default=dt.datetime.utcnow)
+    modified_at = Column(db.DateTime, default=datetime.utcnow, onupdate=datetime.utcnow,
+                         nullable=False)
+    modified_by_id = reference_col('users', nullable=False)
+    modified_by = relationship('User', foreign_keys=modified_by_id, uselist=False)
```

Comment:
For my own understanding: Is the `uselist` param needed? Will `modified_by` be a list of `User` otherwise?

Reply:
Yes, methinks. But let me try it one more time to verify...

Reply:
@mxtthias No, I was wrong.

```reST

        :param uselist:
          a boolean that indicates if this property should be loaded as a
          list or a scalar. In most cases, this value is determined
          automatically by :func:`.relationship` at mapper configuration
          time, based on the type and direction
          of the relationship - one to many forms a list, many to one
          forms a scalar, many to many is a list. If a scalar is desired
          where normally a list would be present, such as a bi-directional
          one-to-one relationship, set :paramref:`~.relationship.uselist` to
          False.

          The :paramref:`~.relationship.uselist` flag is also available on an
          existing :func:`.relationship` construct as a read-only attribute,
          which can be used to determine if this :func:`.relationship` deals
          with collections or scalar attributes::

              >>> User.addresses.property.uselist
              True

          .. seealso::

              :ref:`relationships_one_to_one` - Introduction to the "one to
              one" relationship pattern, which is typically when the
              :paramref:`~.relationship.uselist` flag is needed.

```

SQLAlchemy does this well automatically. No point in adding it everywhere manually. Removing it.